src/router.tsx

```diff
@@ -32,6 +33,14 @@ const router = createBrowserRouter(
     <>
       <Route path="confirmations/:token" element={<Confirmation />} />
       <Route path="password_resets/:token" element={<PasswordResets />} />
+      <Route
```

Comment:
I just tested this in preview, it seems it doesn't pass along the token param unfortunately. Sorry for the flip flop, but it may be best to just update the route paths as you did initially.